sample/Kmd.Logic.Gateway.Automation.Sample/AppConfiguration.cs

```diff
@@ -7,5 +8,7 @@ internal class AppConfiguration
         public LogicTokenProviderOptions TokenProvider { get; set; } = new LogicTokenProviderOptions();
 
         public GatewayOptions Gateway { get; set; } = new GatewayOptions();
+
+        public string FolderPath { get; set; }
```

Comment:
Shouldn't it be set by the user as an input parameter?

Reply:
yes it has been passed on args parameters